src/panels/lovelace/special-rows/hui-text-row.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  css,
+  CSSResult,
+} from "lit-element";
+
+import { LovelaceRow, TextConfig } from "../entity-rows/types";
+
+import "../../../components/ha-icon";
+
+@customElement("hui-text-row")
+class HuiTextRow extends LitElement implements LovelaceRow {
+  @property() private _config?: TextConfig;
+
+  public setConfig(config: TextConfig): void {
+    if (!config || !config.name || !config.text) {
+      throw new Error("Invalid Configuration: 'name' and 'text' required");
+    }
+
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config) {
+      return html``;
+    }
+
+    return html`
```

Comment:
Should we just use the generic row here to keep it the same as the others? Lest maintenance if it changes?

Reply:
generic row relies on an entity.

Reply:
Ah didnt realize